use_cases/eluc/app/requirements.txt

```diff
@@ -1,4 +1,4 @@
-dash==2.10.2
+dash==2.15
```

Comment:
Needed to update dash to get functionality to have the sliders show as percentages